src/python_testing/TC_OpstateCommon.py

```diff
@@ -122,7 +122,7 @@ def send_raw_manual_or_pipe_command(self, command):
             self.write_to_app_pipe(command)
             time.sleep(0.1)
         else:
-            input("Press Enter when done.\n")
+            self.wait_for_user_input(prompt_msg="Press Enter when ready.\n")
```

Comment:
can you please add the test step text in here? Otherwise I'm not sure quite what the TH shows.

Reply:
It's unclear what the test step is based on the code I'm looking at, since this actually seems to vary.

The method send_raw_manual_or_pipe_command only passes a command as an argument (which is a dictionary containing the following keys: Name, Device, Operation, and Param, all of which are strings.)

The method send_raw_manual_or_pipe_command is wrapped by send_manual_or_pipe_command -- nothing seems to call it directly (based on a grep search). send_manual_or_pipe_command sends these arguments in a JSON format.

send_manual_or_pipe_command is called multiple times, but the steps differ. (For example, one step is "# STEP 6a: Manually put the device in the Stopped(0x00) operational state", another is "# STEP 6c: Manually put the device in the Running(0x01) operational state", etc., and all of these are indicated in _comments_ before this method is called.) There doesn't seem to be anything that prints the steps -- they just call self.step(int) beforehand.

There might have to be a non-trivial rework of this file to improve this method send_raw_manual_or_pipe_command. I can do that if needed, but I don't want to jump the gun without a prior discussion.

Reply:
nah, let's just leave this for now then. If the RVC folks want to re-work it after seeing it in the TH, they can handle it themselves.

Reply:
That leaves me with a couple of questions:

1. Should an issue be file to address the needed changes?
2. Should I revert this one-line change or keep it?

Reply:
This is fine. RVC can open an issue if they find it's a problem when they go to test.